ui/components/app/transaction-alerts/transaction-alerts.js

```diff
@@ -16,16 +16,28 @@ import { isSuspiciousResponse } from '../../../../shared/modules/security-provid
 import BlockaidBannerAlert from '../security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert';
 ///: END:ONLY_INCLUDE_IN
 import SecurityProviderBannerMessage from '../security-provider-banner-message/security-provider-banner-message';
+import { getNativeCurrency } from '../../../ducks/metamask/metamask';
 
 const TransactionAlerts = ({
   userAcknowledgedGasMissing,
   setUserAcknowledgedGasMissing,
   txData,
+  title,
 }) => {
   const { estimateUsed, hasSimulationError, supportsEIP1559, isNetworkBusy } =
     useGasFeeContext();
   const pendingTransactions = useSelector(submittedPendingTransactionsSelector);
   const t = useI18nContext();
+  const nativeCurrency = useSelector(getNativeCurrency);
+
+  const currentTokenSymbol = title ? title.split(' ')[1] : nativeCurrency;
+  const currentTokenAmount = title
+    ? title.split(' ')[0]
+    : txData.txParams.value;
+  // isSendingZero is true when either sending native tokens where the value is in txParams
+  // or sending tokens where the value is in the title
```

Comment:
I don't know if we should be relying upon the title here, it feels like a change to our UX could cause a functionality issue. instead why don't we use the txParams.data field to get the token amount if its not a eth send?

Reply:
Thanks @brad-decker ! 🙏  I pushed an update to get the the amount from txParams and get the token symbol from tokenSymbol prop instead of title, 
Lemme know if anything! 🙏